modules/db_instance/main.tf

```diff
@@ -17,6 +17,8 @@ resource "aws_db_instance" "this" {
   password = "${var.password}"
   port     = "${var.port}"
 
+  replicate_source_db = "${var.source_db}"
```

Comment:
use variable and value the same - "replicate_source_db" (not "source_db")

Reply:
updated: replicate_source_db = "${var.replicate_source_db}"